styles.css

```diff
@@ -436,7 +436,24 @@ button:active {
 
 @media only screen and (min-width: 768px) {
   a.navbar-logo {
-    margin-left: -1%;
+    left: -567%;
+  }
+
+  #open,
+  #close {
+    display: none;
+  }
+
+  .menu-mobile {
+    opacity: 1;
+    visibility: visible;
+    background: transparent;
+    backdrop-filter: none;
+  }
+
+  .menu-mobile a {
+    color: #344563;
```

Comment:
You did well with the implementation of this project, kindly implement the following changes so your design looks better and attractive 😄 

- Please I'd suggest you add the blurred background of the mobile popup, as you are expected to follow the design as closely as possible 
- I'd also suggest you work on the margin or positioning of the `mobile menu` and  `close button`. it looks far away from it's expected position 

Your design :
![image](https://github.com/migcm06/miguel-portfolio/assets/20537712/854b5e42-af8a-40bb-a96f-04e441d73794)

Figma design :
![image](https://github.com/migcm06/miguel-portfolio/assets/20537712/fab4faf0-0c4d-4cdd-bd8a-d6d20c96352d)